examples/AN00162_i2s_loopback_demo/src/main.xc

```diff
@@ -18,16 +18,17 @@ on tile[0]: clock bclk = XS1_CLKBLK_2;
 on tile[0]: port p_i2c = XS1_PORT_4A;
 on tile[0]: port p_gpio = XS1_PORT_8C;
 
-#define SAMPLE_FREQUENCY 48000
-#define MASTER_CLOCK_FREQUENCY 24576000
+#define SAMPLE_FREQUENCY (48000)
+#define MASTER_CLOCK_FREQUENCY (24576000)
+#define DATA_BITS (32)
 
-#define CS5368_ADDR           0x4C // I2C address of the CS5368 DAC
-#define CS5368_GCTL_MDE       0x01 // I2C mode control register number
-#define CS5368_PWR_DN         0x06
+#define CS5368_ADDR           (0x4C) // I2C address of the CS5368 DAC
```

Comment:
Brackets aren't needed for #define constants. They don't harm, but are not necessary.
https://stackoverflow.com/questions/28794578/why-when-i-use-define-for-int-i-need-to-wrap-them-in-brackets

They are very much needed for any expressions however